vector-im/element-android
# Space Switching: Space Invites

## Type of change


## Content

Adds invites to space list

## Motivation and context

Closes https://github.com/vector-im/element-android/issues/6583

## Screenshots / GIFs



| Light | Dark | Black |
|-|-|-|
| <img width=300 src="https://user-images.githubusercontent.com/20701752/186398534-83a06632-eec6-42cc-9856-3a3bcdc64e2a.png" /> | <img width=300 src="https://user-images.githubusercontent.com/20701752/186398652-ee4959d9-e67d-41f9-b80c-d7440da16d12.png" /> | <img width=300 src="https://user-images.githubusercontent.com/20701752/186398764-9bb8a78e-2251-43d0-8e3e-e53b99dd3c06.png" /> |

## Tests

- Be invited to a space (without accepting the invitation)
- Open the space list and see the invite at the bottom
- Click on the invite and see that you can respond to the invite


## Tested devices

- OS version(s): Android 12

## Checklist
